sdks/python/apache_beam/typehints/schemas.py

```diff
@@ -226,6 +226,15 @@ def option_from_runner_api(
       schema_registry=schema_registry).option_from_runner_api(option_proto)
 
 
+def schema_field(
+    name: str, field_type: Union[schema_pb2.FieldType,
```

Comment:
do we have enough tests for core.py and schemas.py?

Reply:
They should be quite well covered.

Reply:
I am fine if the existing tests cover the new code changes well.